src/app/provider.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";//use client es para solicionar posibles errores que las librerias no soportan en modo servidor
```

Comment:
Keep this kind of notes for yourself.

@geoperez what about this 'use client' thing?

Reply:
Recommended for next.js